internal/cmd/env_diff.go

```diff
@@ -43,13 +49,15 @@ func NewEnvDiff() *EnvDiff {
 func BuildEnvDiff(e1, e2 Env) *EnvDiff {
 	diff := NewEnvDiff()
 
+	ignoredKeys := NewIgnoredKeys(e2)
```

Comment:
Let's pass the IngoredKeys instance as a function argument so it can be passed from the config object instead.